stern/tail.go

```diff
@@ -96,13 +96,22 @@ var colorList = [][2]*color.Color{
 	{color.New(color.FgHiRed), color.New(color.FgRed)},
 }
 
-func determineColor(podName string) (podColor, containerColor *color.Color) {
-	hash := fnv.New32()
-	_, _ = hash.Write([]byte(podName))
-	idx := hash.Sum32() % uint32(len(colorList))
+func determineColor(podName, containerName string, diffContainer bool) (podColor, containerColor *color.Color) {
+	podHash := fnv.New32()
+	_, _ = podHash.Write([]byte(podName))
+	podIdx := podHash.Sum32() % uint32(len(colorList))
+	colors := colorList[podIdx]
+
+	if diffContainer {
+		containerHash := fnv.New32()
+		_, _ = containerHash.Write([]byte(containerName))
+		containerIdx := containerHash.Sum32() % uint32(len(colorList))
+		containerColor = colorList[containerIdx][1]
+	} else {
+		containerColor = colors[1]
+	}
 
-	colors := colorList[idx]
-	return colors[0], colors[1]
+	return colors[0], containerColor
```

Comment:
How about moving the index calculation into a separate function?

```go
func determineColor(podName, containerName string, diffContainer bool) (podColor, containerColor *color.Color) {
	colors := colorList[colorIndex(podName)]
	if diffContainer {
		return colors[0], colorList[colorIndex(containerName)][1]
	}
	return colors[0], colors[1]
}

func colorIndex(name string) uint32 {
	hash := fnv.New32()
	_, _ = hash.Write([]byte(name))
	return hash.Sum32() % uint32(len(colorList))
}
```

Reply:
@tksm Thanks, applied your suggestion.